packages/sync/src/connect-webrtc.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+// @ts-ignore
+import { WebrtcProvider } from 'y-webrtc';
+
+/** @typedef {import('./types').ObjectType} ObjectType */
+/** @typedef {import('./types').ObjectID} ObjectID */
+/** @typedef {import('./types').CRDTDoc} CRDTDoc */
+
+/**
+ * Connect function to the IndexedDB persistence provider.
+ *
+ * @param {ObjectID}   objectId   The object ID.
+ * @param {ObjectType} objectType The object type.
+ * @param {CRDTDoc}    doc        The CRDT document.
+ *
+ * @return {Promise<() => void>} Promise that resolves when the connection is established.
+ */
+export function connectWebRTC( objectId, objectType, doc ) {
+	const docName = `${ objectType }-${ objectId }`;
+	new WebrtcProvider( docName, doc, {
```

Comment:
We are creating a connection per document, and the document has "edited", and "base" (each entity has two documents), so with 3 users collaborating, and assuming the site editor loads an average of 7 entities (it can easily be more) we have 7*2*3= 42 socket connections. I'm not sure if there are limits on the number of websocket connections the browser can keep open but I guess it is something we should keep an eye on.

Reply:
From what I've read on yjs, that seems like a common approach but of course, it's something we should check.